framework/validators/UniqueValidator.php

```diff
@@ -133,7 +145,24 @@ public function validateAttribute($model, $attribute)
         }
 
         if ($exists) {
-            $this->addError($model, $attribute, $this->message);
+            if (is_array($targetAttribute)) {
+                $attributeCombo = [];
+                $valueCombo = [];
+                foreach ($targetAttribute as $k => $v) {
```

Comment:
No problem, but it would be totally inconsistent to have here full names and just a few lines above in the same method in the similar loop I copy pasted to have short names breaking your core code style.